event/monitoring.go

```diff
@@ -74,15 +74,17 @@ const (
 
 // strings for pool command monitoring types
 const (
-	ConnectionClosed   = "ConnectionClosed"
 	PoolCreated        = "ConnectionPoolCreated"
+	PoolReady          = "ConnectionPoolReady"
```

Comment:
I'm not sure why the existing pool event constants are named different than the strings, but changing them would be a backward-breaking change since they're exported and not in the `x/` directory. I decided to stick with the existing pattern for consistency, even tho the pattern is weird.

Reply:
Sounds good.